packages/components/src/form-token-field/style.scss

```diff
@@ -152,6 +152,7 @@
 	color: $dark-gray-500;
 	line-height: 10px;
 	overflow: initial;
+	height: 24px;
```

Comment:
I would set this as `height: auto` instead so it's a little more resilient to change down the line.